exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/test_slc.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>"Now you <strong>can</strong> ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>fixed</p>